test/helpers/kubectl.go

```diff
@@ -1343,3 +1343,18 @@ func (kub *Kubectl) WaitCEPRevisionIncrease(podName, podNamespace string, oldRev
 	return WithTimeout(body, "CEP revision did not increase after specified timeout", &TimeoutConfig{Timeout: HelperTimeout})
 
 }
+
+// WaitForCEPToExist waits for the endpoint model representing the CiliumEndpoint
+// for pod podName in the specified namespace podNamespace to be non-nil. If it
+// is still nil after a specified timeout, returns an error.
+func (kub *Kubectl) WaitForCEPToExist(podName, podNamespace string) error {
```

Comment:
I would change here the order, podNamespace and podname. It's the way that we follow in all the test.